### PR TITLE
fix: Proper fix for duplicated container content events

### PR DESCRIPTION
**Pull request by magicus**
Proper fix for duplicated container content events when using the container query.

This is a prerequisite for dynamic queries.

### CODE REVIEW

File: common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java [hunk 1]
```diff
@@ -60,6 +60,22 @@ public static List<ItemStack> getAccessories(Player player) {
...
+    public static boolean isItemListsEqual(List<ItemStack> firstItems, List<ItemStack> secondItems) {
+        if (firstItems.size() != secondItems.size()) return false;
+
+        for (int i = 0; i < firstItems.size(); i++) {
+            var newItem = firstItems.get(i);
+            var oldItem = secondItems.get(i);
+            if (!newItem.getItem().equals(oldItem.getItem())
+                    || newItem.getDamageValue() != oldItem.getDamageValue()
+                    || newItem.getCount() != oldItem.getCount()
+                    || !ItemStack.tagMatches(oldItem, newItem)) {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java:72]: Doesn't ItemStack#equals do this which means firstItems.equals(secondItems) does this?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java:72]: ItemStack doesn't implement equals... Okay.

**Review by kristofbolyai — APPROVED** [review]
Looks good!

File: common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java:72]: Yep, I noticed that. Hence this method. I hope that comparing item type, damage, count and lore that I capture everything that is relevant.

**Pull request merged**